doc/manual/chapters/translation.tex

```diff
@@ -1,10 +1,364 @@
+	stmt6
+// ...
+endEither:
+	\end{lstlisting} \\
+	\hline
+	Assignment & \lstinline[language=pcal]|x := exp;| & Supported; compiled as expected \\
+	\hline
+	Multiple variable assignment & \lstinline[language=pcal]!x := y || y := x + y;! & Supported; compiled as multiple assignment in Go \\
+	\hline
+	Return statement & \lstinline[language=pcal]|return;| & Supported; compiled as expected \\
+	\hline
+	Skip statement & \lstinline[language=pcal]|skip;| & Supported; compiled to nothing \\
+	\hline
+	Call statement & \lstinline[language=pcal]|call procedure1(arg1, arg2);| & Supported; compiled as expected \\
+	\hline
+	Macro call & \lstinline[language=pcal]|macro1(arg1, arg2);| & Supported; expanded during compilation \\
+	\hline
+	With statement &
+	\begin{lstlisting}[language=pcal]
+with (x = exp1, y \in exp2) {
+	body;
+}
+	\end{lstlisting} &
+	Supported; compiled as variable assignment with fresh names \\
+	\hline
+	Print statement &
+	\lstinline[language=pcal]|print exp;| & Supported; compiled as expected \\
+	\hline
+	Assert statement & \lstinline[language=pcal]|assert condition;| &
+	Compiled as
+	\begin{lstlisting}[language=golang]
+if !condition {
+	panic("condition");
+}
+	\end{lstlisting} \\
+	\hline
+	Await statement & \lstinline[language=pcal]|await condition;| &
+	Compiled as
+	\begin{lstlisting}[language=golang]
+awaitLabel:
+	if !condition {
+		goto awaitLabel
+	}
+	\end{lstlisting} \\
+	\hline
+	Goto statement & \lstinline[language=pcal]|goto label;| & Supported; compiled as expected \\
+	\hline
+	Single process algorithm &
+	\begin{lstlisting}[language=pcal]
+--algorithm Algo {
+  variables x = exp1, y \in exp2;
+  {
+  	body;
+  }
+}
+	\end{lstlisting} &
+	Supported; compiled as a single-threaded single-process Go program \\
+	\hline
+	Multiprocess algorithm &
+	\begin{lstlisting}[language=pcal]
+--algorithm Algo {
+  variables x = exp1, y = exp2;
+  process (P \in exp3)
+  variables local = exp4;
+  {
+  	body;
+  }
+}
+	\end{lstlisting} &
+	Supported; compiled with various strategies configured by the user \\
+	\hline
+	\caption{PlusCal constructs}
+	\label{table:pcal-constructs}
+\end{longtable}
+\end{center}
+
+Below are the TLA+ constructs. Note that PGo makes liberal use of temporary variables to compile complex TLA+ constructs.
+
+\begin{center}
+\lstset{style=tabularstyle}
+\renewcommand{\arraystretch}{1.5}
+\begin{longtable}{ || m{0.25\textwidth} | m{0.33\textwidth} | m{0.36\textwidth} || }
+	\hline
+	\textbf{TLA+ feature} & \textbf{Example code} & \textbf{PGo support} \\
+	\hline\hline
+	Function call &
+	\begin{lstlisting}[language=pcal]
+x[exp1]
+\* or
+x[exp1, exp2, exp3]
+\* or
+x[<<exp1, exp2, exp3>>]
+\* or
+x[[field1 |-> e1, field2 |-> e2]]
+	\end{lstlisting} &
+	Supported; compiled code dependent on the type of \lstinline[language=pcal]|x| (the function) \\
+	\hline
+	Binary operator call & \lstinline[language=pcal]|x /\ y = z + 1| & Supported; compiled as expected \\
+	\hline
+	Function (as struct literal) &
```

Comment:
I think these are typically called "Records" in TLA+ documents, so we better stick to that naming.